tools/syz-declextract/README.md

```diff
@@ -0,0 +1,41 @@
+# syz-declextract
+## Linux Kernel (For testing purposes)
+```
+export KERNEL=$PWD/linux-stable
+git clone \
```

Comment:
We generally don't do 80 columns (don't tend to print our code on punch cards and teletypes :)).
For Go code we cap line length at 120 columns, but for other code we just use best judgement of what's readable.

Folding such lines that are a bit longer than 80 columns can make code more readable.

Reply:
Done. I agree it's more readable now!